.github/ISSUE_TEMPLATE.md

```diff
@@ -1,25 +1,48 @@
-<!--
-
-** Please read the guidelines below. **
-
-GitHub issues are reserved for bug reports.
-The best place to ask general questions or ask for community support is
-[on our Discourse](https://discourse.snowplowanalytics.com/).
-
--->
-
-**Project**: <!-- e.g. Stream Enrich, Scala Stream Collector -->
-
-**Version**:
-
-**Expected behavior**:
-
-**Actual behavior**:
-
-**Steps to reproduce**:
-
-<!-- Please try to be as detailed as possible so that we can reproduce and fix the issue
-as quickly as possible. -->
- 1.
- 2.
-
+/!\ IMPORTANT /!\
```

Comment:
When I review it GH tells me we're using an old approach: https://docs.github.com/en/free-pro-team@latest/github/building-a-strong-community/about-issue-and-pull-request-templates